### DISCUSSION TITLE
Organizationページのホームに「About」が表示されていたほうがいいのでは？

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#432] [要望・アイデア・不具合]
先日、Organizationページがアップデートされましたね。

[Organizationページをアップデートします - Qiita Blog](https://blog.qiita.com/organization-page-update-2023-2-9/)

全体的に良いアップデートだと思うのですが、「About」はホームに表示されていたほうがいいのではないかと思います。
なぜなら、「About」にはMarkdownも使えることから、結構気合を入れて書いている組織も多そう（[Qiita株式会社](https://qiita.com/organizations/qiita-inc/profile)もそうでしょう）だからです。

もっとも、「よりメンバーの活動がわかりやすく、組織の情報も網羅的に確認しやすい」というコンセプトにのっとって、現在ホームに表示されている「人気の記事」や「Contributors」を目立たせたいという気持ちもわかります。
GitHubにはOrganizationページにREADMEを表示する機能がありますが、READMEの下にピンしたリポジトリ、リポジトリ一覧などが並び、サイドにはメンバーのアカウントやよく使われる言語などが表示されています。

https://github.com/github

QiitaのOrganizationページでも、「About」をホームに表示しつつ、「よりメンバーの活動がわかりやすく、組織の情報も網羅的に確認しやすい」というコンセプトを実現できるとよさそうです。

**Comment by kyntk** [comment]
@munierujp ご意見ありがとうございます🙏
実は、Organizationページをアップデートする中で、ホームにAboutを入れることも検討していました。

その中で、Aboutをホームに入れなかった理由は2点あります。

- Organization名の下にあるdescriptionと役割が似ているため
- Aboutの文量が多いとメンバーの活動が目立たなくなること

## Organization名の下にあるdescriptionと役割が似ていることについて
Organizaiton名の下にあるdescriptionは、Aboutの要約になることが多いと考えています。　そのため、ユーザーにとってOrganizaitonの情報が伝わりやすいのはdiscriptionであると考えています。

## Aboutの文量が多いとメンバーの活動が目立たなくなることについて
Aboutは、Organizationごとに文量が違います。
Aboutの文量が多い場合、全文を載せるか ある程度で省略する必要があります。全文を載せると、他のセクションが目立たなくなり、メンバーの活動の印象が薄くなります。また、省略すると、Organizationが伝えたいことが伝わらなくなります。 

この2点から、Aboutをホームに入れませんでした。

今後はOrganization名の下にあるdescriptionを要約として使っていただき、AboutはよりOrganizationについて知りたい際に使っていただけると幸いです。
